### PR TITLE
 the field on the student  and the price they pay, during registratio…

### PR DESCRIPTION
**Pull request by hebaabbadi**
…n are done

### CODE REVIEW

File: components/forms/user-form.tsx [hunk 1]
```diff
@@ -56,31 +58,63 @@ export default function UserForm({
...
+            className="small mb-1"
+            htmlFor="inputPayment"
+            {...register('Payment')}
+          >
+            Payment
+          </label>
+          <input
+            {...register('Payment')}
+            className="form-control"
+            type="text"
```
Comment (mbcube) [inline components/forms/user-form.tsx:101]: type devrais pas etre text, c'est des numero seulement

**Review by mbcube — CHANGES_REQUESTED** [review]

File: components/forms/user-form.tsx [hunk 2]
```diff
@@ -96,9 +96,9 @@ export default function UserForm({
             Payment
           </label>
           <input
-            {...register('Payment')}
+            {...register('payment')}
             className="form-control"
-            type="text"
+            type="number"
             placeholder="Enter the Payment"
             {...register('Payment', {
```
Comment (mbcube) [inline components/forms/user-form.tsx:103]: you're registering twice

**Review by mbcube — CHANGES_REQUESTED** [review]

File: pages/users/[userId].tsx [hunk 3]
```diff
@@ -38,6 +38,8 @@ export default function EditUser() {
         email: formData.email,
         displayName: formData.displayName,
         userType: formData.userType,
+        Payment: formData.Payment,
```
Comment (mbcube) [inline pages/users/[userId].tsx:41]: lowercase

Comment (mbcube) [reply pages/users/[userId].tsx:41]: payment: formData.payment

File: pages/users/index.tsx [hunk 4]
```diff
@@ -74,9 +76,13 @@ export default function UserList() {
                                       </div>
                                     </td>
                                     <td>{user.email}</td>
+                                    {UserType.Student &&
+                                      userType == UserType.Student && (
+                                        <td>{user.Payment}</td>
+                                      )}
```
Comment (mbcube) [inline pages/users/index.tsx:82]: lowercase: user.payment

File: pages/users/new.tsx [hunk 5]
```diff
@@ -1,57 +1,61 @@
...
+    return response.data.uid
   }
 
   async function createUserDocument(
     uid: string,
     email: string,
     displayName: string,
-    userType: UserType
+    userType: UserType,
+    Payment: null
```
Comment (mbcube) [inline pages/users/new.tsx:51]: lowercase:  payment

**Review by mbcube — CHANGES_REQUESTED** [review]

File: pages/users/[userId].tsx [hunk 6]
```diff
@@ -38,6 +38,8 @@ export default function EditUser() {
         email: formData.email,
         displayName: formData.displayName,
         userType: formData.userType,
+        Payment: formData.payment,
```
Comment (mbcube) [inline pages/users/[userId].tsx:41]: `payment : formData.payment`

-.-

File: pages/users/new.tsx [hunk 7]
```diff
@@ -1,57 +1,61 @@
...
       await createUserDocument(
         uid,
         formData.email,
         formData.displayName,
-        formData.userType
-      );
-      toast.success(`Account Created`);
-      router.push(`/users`);
+        formData.userType,
+        formData.Payment
```
Comment (mbcube) [inline pages/users/new.tsx:26]: `formData.payment`

-.-

File: pages/users/new.tsx [hunk 8]
```diff
@@ -1,57 +1,61 @@
...
   }
 
   async function createUser(email: string): Promise<string> {
-    const response = await axios.post("/api/firebase-api", {
+    const response = await axios.post('/api/firebase-api', {
       authUser: auth.currentUser,
       userType: user?.userType,
       email,
-    });
+      Payment: null,
```
Comment (mbcube) [inline pages/users/new.tsx:40]: `payment: null`

-.-

**Review by mbcube — CHANGES_REQUESTED** [review]

**Review by mouadbouras — APPROVED** [review]

**Review by mouadbouras — APPROVED** [review]

**Review by mbcube — APPROVED** [review]

**Pull request merged**